views/user.ejs

```diff
@@ -1,40 +1,83 @@
+				</div>
+				<div class="col-sm-9">
+					<div class="d-flex flex-column" style="height: 100%">
+						<div class="d-flex bigfont" style="height: 100%; align-items: center"><%= user.name%> "<%= user.username%>" <%= user.lastName%></div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+	<div class="row justify-content-center p-1" style="margin-top: 20px">
+		<h1>Utwórz nową grupę</h1>
+		<form action="/group/add" method="POST">
+			<label for="groupName">Nazwa nowej grupy:</label><br>
+			<input type="text" name="groupName" class="borderHighlight" required><br>
+			<label for="description">Opis nowej grupy:</label><br>
+			<input type="text" name="description" class="borderHighlight" required>
+			<button class="btn btn-outline-primary">Utwórz</button>
+		</form>
+	</div>
+	<div class="row justify-content-center p-1" style="margin-top: 20px">
+		<h1>Dołącz do grupy</h1>
+		<form action="/user/<%-currentUser._id%>/joinGroup" method="POST">
+			<label for="entryCode">Podaj kod grupy:</label><br>
+			<input type="text" name="entryCode" class="borderHighlight" required>
+			<button class="btn btn-outline-primary">Dołącz</button>
+		</form>
+	</div>
+	<div class="row justify-content-center p-1" style="margin-top: 10px">
+		<h1>Grupy do których należysz:</h1>
+		<%for(let i = 0; i < groupsBelongedTo.length; i++){%>
+		<div class="row borderHighlight m-2 p-2" style="width: 90%">
+			<div class="col-sm-1">
+				<img src="https://wiki.dave.eu/images/4/47/Placeholder.png" class="img-thumbnail rounded-circle" alt="img_user">
+			</div>
+			<div class="col-sm-9">
+				<div class="d-flex flex-column" style="height: 100%">
+					<div class="d-flex" style="height: 100%; align-items: center"><%= groupsBelongedTo[i].groupName%></div>
+				</div>
+			</div>
+			<div class="col-sm-2">
+				<div class="d-flex flex-column center" style="height: 100%">
+					<a class="nav-link active" aria-current="page" href="/group/<%= groupsBelongedTo[i]._id%>">
+						<button type="button" class="btn btn-outline-primary">Dołącz do spotkania</button>
+					</a>
+				</div>
+			</div>
+		</div>
+		<%};%>
+	</div>
+	<div class="row justify-content-center p-1" style="margin-top: 10px">
+		<h1>Grupy które posiadasz:</h1>
+		<%for(let i = 0; i < ownedGroups.length; i++){%>
+		<div class="row borderHighlight m-2 p-2" style="width: 90%">
+			<div class="col-sm-1">
+				<img src="https://wiki.dave.eu/images/4/47/Placeholder.png" class="img-thumbnail rounded-circle" alt="img_user">
+			</div>
+			<div class="col-sm-9">
+				<div class="d-flex flex-column" style="height: 100%">
+					<div class="d-flex" style="height: 100%; align-items: center"><%= ownedGroups[i].groupName%></div>
+				</div>
+			</div>
+			<div class="col-sm-2">
+				<div class="d-flex flex-column center" style="height: 100%">
+					<a class="nav-link active" aria-current="page" href="/group/<%= ownedGroups[i]._id%>">
+						<button type="button" class="btn btn-outline-primary">Modyfikuj grupę</button>
```

Comment:
Może zamiast 'Modyfikuj grupę' dać wyświetl grupę. To lepiej opisze działanie przycisku.